sdk/documenttranslator/ai-document-translator-rest/review/ai-document-translator.api.md

```diff
@@ -4,738 +4,812 @@
-export interface GetDocumentsStatusQueryParamProperties {
+export interface DocumentTranslationGetDocumentsStatusQueryParamProperties {
     $maxpagesize?: number;
-    $orderBy?: string[];
+    $orderBy?: Array<string>;
     $skip?: number;
     $top?: number;
-    createdDateTimeUtcEnd?: Date;
-    createdDateTimeUtcStart?: Date;
-    ids?: string[];
-    statuses?: string[];
+    createdDateTimeUtcEnd?: Date | string;
+    createdDateTimeUtcStart?: Date | string;
+    ids?: Array<string>;
+    statuses?: Array<string>;
 }
 
 // @public (undocumented)
-export interface GetDocumentStatus {
-    get(options?: GetDocumentStatusParameters): Promise<GetDocumentStatus200Response | GetDocumentStatus401Response | GetDocumentStatus404Response | GetDocumentStatus429Response | GetDocumentStatus500Response | GetDocumentStatus503Response>;
-}
-
-// @public (undocumented)
-export interface GetDocumentStatus200Headers {
-    "retry-after"?: string;
+export interface DocumentTranslationGetDocumentStatus200Headers {
+    "retry-after"?: number;
     etag?: string;
 }
 
 // @public
-export interface GetDocumentStatus200Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus200Response extends HttpResponse {
     // (undocumented)
-    body: DocumentStatus;
+    body: DocumentStatusOutput;
     // (undocumented)
-    headers: RawHttpHeaders & GetDocumentStatus200Headers;
+    headers: RawHttpHeaders & DocumentTranslationGetDocumentStatus200Headers;
     // (undocumented)
     status: "200";
 }
 
 // @public
-export interface GetDocumentStatus401Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus401Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "401";
 }
 
 // @public
-export interface GetDocumentStatus404Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus404Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "404";
 }
 
 // @public
-export interface GetDocumentStatus429Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus429Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "429";
 }
 
 // @public
-export interface GetDocumentStatus500Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus500Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "500";
 }
 
 // @public
-export interface GetDocumentStatus503Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus503Response extends HttpResponse {
```

Comment:
have we enabled enableOperationGroup for this service? or is there duplicate operation name ?

Reply:
no it is generated before we support enableOperationGroup